src/game/client/c_playerresource.cpp

```diff
@@ -114,7 +114,22 @@ void C_PlayerResource::OnDataChanged(DataUpdateType_t updateType)
 			player_info_t info;
 			if ( IsConnected( slot ) && engine->GetPlayerInfo( slot, &info ) && !info.fakeplayer && !info.ishltv && !info.isreplay )
 			{
-				playerIDs.AppendFormat( "%s%08x", playerIDs.Length() ? "," : "", info.friendsID );
+				int index = engine->GetPlayerForUserID( info.userID );
+				int score = g_PR->GetPlayerScore( index );
+				CBasePlayer* pPlayer = UTIL_PlayerByIndex( index );
+				if ( pPlayer )
+				{
+					if ( playerIDs.Length() )
+					{
+						playerIDs.Append( ',' );
+					}					
+					for ( const char *psz = pPlayer->GetPlayerName(); *psz; psz++ )
+					{
+						playerIDs.AppendFormat( "%02x", *psz );
+						playerIDs.Append('-');
```

Comment:
Since the bytes are each 2 hex characters, we don't need separators between the bytes.

```suggestion
```